src/app/app.component.html

```diff
@@ -1,7 +1,11 @@
-<alm-app-header></alm-app-header>
-
-<main class="app">
-  <router-outlet></router-outlet>
+<header class="padding-bottom-10" >
```

Comment:
The problem with the header tag being in the alm-app-header is that it effectively becomes a child of the alm-app-header (effectively a div from a layout perspective). The goal is to always to reduce the amount markup required and separate 'layout' concerns (i.e semantic header) with 'content' concerns (contents of the alm-app-header element). This results in more style rules needed which slow things down from a rendering perspective.